ciao_contrib/runtool.py

```diff
@@ -2760,7 +2760,7 @@ def make_tool(toolname):
 
 parinfo['csmooth'] = {
     'istool': True,
-    'req': [ParValue("infile","f","input file name",None),ParValue("sclmap","f","image of user-supplied map of smoothing scales",None),ParValue("outfile","f","output file name",None),ParValue("outsigfile","f","output significance image",'.'),ParValue("outsclfile","f","output scales [kernel sizes] image",'.'),ParSet("conmeth","s","Convolution method.",'fft',["slide","fft"]),ParSet("conkerneltype","s","Convolution kernel type.",'gauss',["gauss","tophat"]),ParValue("sigmin","r","minimal significance, S/N ratio",4),ParRange("sigmax","r","maximal significance, S/N ratio",5,")sigmin",None),ParValue("sclmin","r","initial (minimal) smoothing scale [pixel]",None),ParRange("sclmax","r","maximal smoothing scale [pixel]",None,")sclmin",None),ParSet("sclmode","s","compute smoothing scales or user user-supplied map",'compute',["compute","user"])],
+    'req': [ParValue("infile","f","input file name",None),ParValue("sclmap","f","image of user-supplied map of smoothing scales",None),ParValue("outfile","f","output file name",None),ParValue("outsigfile","f","output significance image",'.'),ParValue("outsclfile","f","output scales [kernel sizes] image",'.'),ParSet("conmeth","s","Convolution method.",'fft',["slide","fft"]),ParSet("conkerneltype","s","Convolution kernel type.",'gauss',["gauss","tophat"]),ParValue("sigmin","r","minimal significance, S/N ratio",4),ParRange("sigmax","r","maximal significance, S/N ratio",5,4,None),ParValue("sclmin","r","initial (minimal) smoothing scale [pixel]",None),ParRange("sclmax","r","maximal smoothing scale [pixel]",None,"INDEF",None),ParSet("sclmode","s","compute smoothing scales or user user-supplied map",'compute',["compute","user"])],
```

Comment:
OOh. This one is an interesting change (thanks to the GH view it makes it more obvious what has changed). So `sigmax.p_min` returns the actual value (after following through the parameter redirection).

Hmmmm.